frame/assistants/subquery/src/graphql/generic/bridge_ethv2_next_collecting_new_message_root_signatures_event.query.graphql

```diff
@@ -1,6 +1,6 @@
 query nextCollectingNewMessageRootSignaturesEvent($block: Int) {
   collectingNewMessageRootSignaturesEvents(
-    orderBy: BLOCK_NUMBER_DESC
+    orderBy: BLOCK_NUMBER_ASC
```

Comment:
ECDSA authorities could only sign on the latest message root, I think we don't need to query message roots out of date.